scripts/check_docs

```diff
@@ -7,7 +7,7 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 source "$SCRIPTS_DIR/common"
 
 # `cargo doc` produces warnings for incorrect paths. These warnings cannot be promoted to errors, so we use grep to detect them.
-if cargo doc --document-private-items --no-deps 2>&1 | grep --ignore-case --quiet --regexp='^warning'; then
+if cargo doc --document-private-items --no-deps 2>&1 | grep --ignore-case --after-context 8 --regexp='^warning'; then
```

Comment:
would it be possible to just print out the entire logs, regardless of any errors / warnings? not just what is captured by grep

Reply:
Yes. I have updated the script.